src/briefcase/platforms/linux/system.py

```diff
@@ -513,7 +531,7 @@ def verify_system_packages(self, app: AppConfig):
             system_installer,
         ) = self._system_requirement_tools(app)
 
-        if system_verify is None:
+        if not (system_verify and self.tools.shutil.which(system_verify[0])):
```

Comment:
I didn't break out the system verifier missing altogether as a different error because this should be exceedingly rare and thus not really worth it.

Reply:
Using `shutil.which` is a nice touch too - much easier to mock tests than `Path.exists()`.